src/pages/blocks/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { CursorPage, CursorPagination } from '@components/commons/CursorPagination'
+import { Head } from '@components/commons/Head'
+import { OverflowTable } from '@components/commons/OverflowTable'
+import { UnitSuffix } from '@components/commons/UnitSuffix'
+import { getWhaleApiClient } from '@contexts/WhaleContext'
+import { Block } from '@defichain/whale-api-client/dist/api/blocks'
+import { formatDistanceToNow } from 'date-fns'
+import { GetServerSidePropsContext, GetServerSidePropsResult, InferGetServerSidePropsType } from 'next'
+import NumberFormat from 'react-number-format'
+
+interface BlocksPageData {
+  blocks: {
+    items: Block[]
+    pages: CursorPage[]
+  }
+}
+
+export default function Blocks ({ blocks }: InferGetServerSidePropsType<typeof getServerSideProps>): JSX.Element {
+  return (
+    <div className='container mx-auto px-4 pt-12 pb-20'>
+      <Head title='Blocks' />
+
+      <div className='flex items-center justify-center pb-6'>
+        <div className='bg-pink-50 rounded p-3'>
+          🚧 Work in progress, this is an early iteration of defiscan.live/blocks. Some features are not available and
+          may not work as expected.
+        </div>
+      </div>
+
+      <h1 className='text-2xl font-semibold'>Blocks</h1>
+
+      <div className='my-6'>
+        <OverflowTable>
+          <OverflowTable.Header>
+            <OverflowTable.Head sticky>HEIGHT</OverflowTable.Head>
+            <OverflowTable.Head>AGE</OverflowTable.Head>
+            <OverflowTable.Head>TRANSACTIONS</OverflowTable.Head>
+            <OverflowTable.Head>MINTER</OverflowTable.Head>
+            <OverflowTable.Head>SIZE (B)</OverflowTable.Head>
+            <OverflowTable.Head>DIFFICULTY</OverflowTable.Head>
+          </OverflowTable.Header>
+
+          {blocks.items.map(block => (
+            <BlockRow block={block} key={block.height} />
+          ))}
+        </OverflowTable>
+      </div>
+      <div className='flex justify-end mt-8'>
+        <CursorPagination pages={blocks.pages} path='/blocks' />
+      </div>
+    </div>
+  )
+}
+
+function BlockRow ({ block }: { block: Block }): JSX.Element {
+  return (
+    <OverflowTable.Row key={block.id}>
+      <OverflowTable.Cell sticky>
+        {block.height}
+        {/* <Link href={{ pathname: `/blocks/${block.id}/transactions` }}> */}
+        {/*  <a>{block.height}</a> */}
+        {/* </Link> */}
+      </OverflowTable.Cell>
+      <OverflowTable.Cell className='whitespace-nowrap'>
+        {formatDistanceToNow(block.medianTime * 1000)} ago
```

Comment:
I think we need to use a `useEffect` hook for this and update the age every 3 seconds, since this changes every second.